research.php

```diff
@@ -107,6 +107,19 @@
 
 <?php
     $pubs = array(
+        array(
+            "author"    => "Fritz Alder, Jo Van Bulck, Frank Piessens, Jan Tobias Mühlberg",
+            "title"     => "Aion: Enabling Open Systems through Strong
+                            Availability Guarantees for Enclaves",
+            "publisher" => "Proceedings of the 28th ACM Conference on Computer and Communications Security (CCS)"
+            "date"      => "2021",
+            "id"        => "ccs21",
+            "src"       => "https://github.com/sancus-tee/sancus-riot",
+            "pdf"       => "https://falder.org/files/paper/2021_aion.pdf",
```

Comment:
minor, but I prefer to upload a ccs21.pdf PDF file here instead of linking externally (I've only done external linking so far for papers without DistriNet authors where we may not have the rights). This may also make it easier for the PDF to get picked up by engines like Google Scholar that recognize the Sancus website :)